tech/languages/ocaml/ocaml-installation.md

```diff
@@ -0,0 +1,50 @@
+---
+title: OCaml
+subsection: ocaml
+section: tech-languages
+order: 1
+version: 1.0
+description: An open source multi-paradigm programming language and toolchain. 
+---
+
+# What is OCaml?
+
+Quoting the [ocaml.org](https://ocaml.org/) website:
+
+> OCaml is an industrial-strength programming language supporting functional, imperative and object-oriented styles.
+
+## OCaml installation
+
+To install the OCaml tools, type on a terminal:
+
+```console
+$ sudo dnf install ocaml
+```
+
+The `ocaml` and `ocamlfind`, `ocamlc` and `ocamlopt` binaries will become available on the system.
```

Comment:
Can you briefly describe what they're for, maybe showcase their basic use? Or at least point users to `--help` (generally), or some alternative.

Reply:
Ok, I will add it.